### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by donavanbecker** [sinusphi/venvipy#18]
```
pip install venvipy      
Collecting venvipy
  Using cached venvipy-0.3.6.tar.gz (312 kB)
  Preparing metadata (setup.py) ... done
Collecting PyQt5==5.15.4
  Using cached PyQt5-5.15.4.tar.gz (3.3 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      Traceback (most recent call last):
        File "/Users/donavanbecker/VenviPy/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/Users/donavanbecker/VenviPy/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/Users/donavanbecker/VenviPy/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 152, in prepare_metadata_for_build_wheel
          whl_basename = backend.build_wheel(metadata_directory, config_settings)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/sipbuild/api.py", line 46, in build_wheel
          project = AbstractProject.bootstrap('wheel',
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/sipbuild/abstract_project.py", line 87, in bootstrap
          project.setup(pyproject, tool, tool_description)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/sipbuild/project.py", line 585, in setup
          self.apply_user_defaults(tool)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-install-512cxqe6/pyqt5_b586f360d1db48dd8b122221955c59e9/project.py", line 63, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/pyqtbuild/project.py", line 70, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/sipbuild/project.py", line 236, in apply_user_defaults
          self.builder.apply_user_defaults(tool)
        File "/private/var/folders/3x/rxxslfy94jv3j4pw0d9bppqm0000gn/T/pip-build-env-_xie1k47/overlay/lib/python3.10/site-packages/pyqtbuild/builder.py", line 69, in apply_user_defaults
          raise PyProjectOptionException('qmake',
      sipbuild.pyproject.PyProjectOptionException
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```

**Comment by sinusphi** [comment]
Thx for submitting this error.

**Comment by donavanbecker** [comment]
I did just that and wasn't able too.

**Comment by sinusphi** [comment]
It was because of missing `pyproject.toml`. It's fixed now and I released v0.3.7 of venvipy.
